src/com/sun/ts/tests/assembly/util/shared/resref/single/ByteArrayDataSource.java

```diff
@@ -27,9 +27,9 @@
 import java.util.*;
 import com.sun.javatest.Status;
 
-import javax.mail.*;
-import javax.activation.*;
-import javax.mail.internet.*;
```

Comment:
@bshannon fyi, you mentioned that you were going to create a pr for mail + activation.  Do you want to review these changes or continue with your own pr for that?

This pr also duplicates the existing transaction pr https://github.com/eclipse-ee4j/jakartaee-tck/pull/174

Reply:
I was planning to make my changes across the entire TCK code base, but if someone
else wants to make the changes for me, I'm ok with that.  :-)  Just let me know whether
to continue or to stop.

If @Tomas-Kraus is only doing part of the work and I need to do the rest, just let me
know when his part if done and what else I need to do.

Reply:
Also, I'm planning to rename any "javamail" or "jmail" directories or files to just "mail".

Reply:
@bshannon I'd like to merge it as it is now and work only on metro tests after that

Reply:
@Tomas-Kraus please do continue with your change to mail or activation and we can let @bshannon know when he can continue on top of your changes.

Reply:
In terms of what is left to do, I think it is something like the following (using ag silver search tool):

ag --ignore .git  -l javax.| xargs sed -i 's/javax\.enterprise\./jakarta\.enterprise\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.faces\./jakarta\.faces\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.inject\./jakarta\.inject\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.interceptor\./jakarta\.interceptor\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.jms\./jakarta\.jms\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.json\./jakarta\.json\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.management.j2ee\./jakarta\.management.j2ee\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.persistence\./jakarta\.persistence\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.resource\./jakarta\.resource\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.security.auth\./jakarta\.security.auth\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.security.enterprise\./jakarta\.security.enterprise\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.security.jacc\./jakarta\.security.jacc\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.servlet\./jakarta\.servlet\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.validation\./jakarta\.validation\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.websocket\./jakarta\.websocket\./g'
ag --ignore .git  -l javax.| xargs sed -i 's/javax\.ws\.rs\./jakarta\.ws\.rs\./g'

Reply:
hmm, maybe not javax.management.j2ee, I think that is pruned

Reply:
Yes, but this patch is now quite huge. I'd like to merge it and start another one.

Reply:
> Yes, but this patch is now quite huge. I'd like to merge it and start another one.
Sorry @Tomas-Kraus , I meant the comment about updating the other packages for Bill Shannon, as he mentioned that he is going to finish making needed changes, after this pr is merged.